src/style.css

```
@@ -0,0 +1,106 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 30px 14px 14px;
+  border: 1px solid hsl(0, 0%, 95%);
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin: 0 5px 45px;
+}
+
+.card__title {
+  font-size: 12px;
+  margin: 0;
+  font-weight: 500;
+  line-height: 15px;
+  margin-bottom: 6px;
+}
+
+.card__code {
+  font-size: 11px;
+  line-height: 13px;
+  margin-bottom: 15px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 18px;
+}
+
+.card__stars {
+  display: flex;
+  align-items: center;
+}
+
+.star {
+  background: url(images/star.svg) no-repeat center;
+  height: 15px;
+  width: 15px;
+  margin-right: 4px;
+}
+
+.card__stars > :nth-child(-n+4),
```

```suggestion
stars--4 > :nth-child(-n+4),
```
and add this class into div with class card__stars

---

don't try to invent something just use your stars block task